get-resource.sh

```diff
@@ -7,6 +7,7 @@ export https_proxy=${https_proxy:-$HTTPS_PROXY}
 
 # Which image should we use
 SNAP=${1:-current-tripleo-rdo}
+IPA_BASEURI=${IPA_BASEURI:-https://images.rdoproject.org/stein/rdo_trunk/$SNAP/}
```

Comment:
maybe change this to a more recent version? Ideally would be master, but a recent stable version is fine too
they're under the centos8 dir

Reply:
What is the most recent stable version? Using that sgtm. I get confused with the OpenStack naming scheme...

Reply:
wallaby is the most recent stable
you can use https://images.rdoproject.org/centos8/wallaby/rdo_trunk/$SNAP/

Reply:
Let's fix this, then this patch is mergeable. The current link with `stein` in it only works because the default URI is never used.

Reply:
updated